srcpkgs/redshift/template

```diff
@@ -14,7 +14,9 @@ license="GPL-3.0-or-later"
 homepage="http://jonls.dk/redshift/"
 distfiles="https://github.com/jonls/redshift/releases/download/v${version}/redshift-${version}.tar.xz"
 checksum=d2f8c5300e3ce2a84fe6584d2f1483aa9eadc668ab1951b2c2b8a03ece3a22ba
+
 build_options="geoclue2"
+build_options_default="geoclue2"
```

Comment:
as geoglue is an optional runtime dependency, please remove the build_option for it.
`redshift.conf.sample` might be handy to have in the package